vars/pipelineDeploy.groovy

```diff
@@ -116,7 +135,8 @@ void call() {
                         }
                         steps {
                             script {
-                                doGitClone('vega', params.VEGA_VERSION)
+                                def repoVars = doGitClone('vega', params.VEGA_VERSION)
+                                env.DOCKER_IMAGE_TAG_HASH = sh (returnStdout: true, script: "echo \"${repoVars.GIT_COMMIT}\"|cut -b1-8").trim()
```

Comment:
We would also use the groovy command: 
`repoVars.GIT_COMMIT?.substring(0, 5))`

Reply:
But I like your solution as well

Reply:
done